stream-video-android-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/call/renderer/ParticipantsSpotlight.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-video-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.video.android.compose.ui.components.call.renderer
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import io.getstream.video.android.compose.ui.components.call.renderer.internal.SpotlightVideoRenderer
+import io.getstream.video.android.core.Call
+import io.getstream.video.android.core.ParticipantState
+
+/**
+ * Renders all the CallParticipants, based on the number of people in a call and the call state.
+ * Also takes into account if there are any screen sharing sessions active and adjusts the UI
+ * accordingly.
+ *
+ * @param call The call that contains all the participants state and tracks.
+ * @param modifier Modifier for styling.
+ * @param isZoomable Decide to this screensharing video renderer is zoomable or not.
+ * @param style Defined properties for styling a single video call track.
+ * @param videoRenderer A single video renderer renders each individual participant.
+ */
+@Composable
+public fun ParticipantsSpotlight(
+    call: Call,
+    modifier: Modifier = Modifier,
+    isZoomable: Boolean = false,
+    style: VideoRendererStyle = SpotlightVideoRendererStyle(),
+    videoRenderer: @Composable (
+        modifier: Modifier,
+        call: Call,
+        participant: ParticipantState,
+        style: VideoRendererStyle,
+    ) -> Unit = { videoModifier, videoCall, videoParticipant, videoStyle ->
+        ParticipantVideo(
+            modifier = videoModifier,
+            call = videoCall,
+            participant = videoParticipant,
+            style = videoStyle,
+        )
+    },
+) {
```

Comment:
It mush be great if we can have a doc for this with named `04-participants-spotlight.mdx` under the path below:

https://github.com/GetStream/stream-video-android/tree/develop/docusaurus/docs/Android/04-ui-components/05-participants